src/cli/utils/sharepoint.go

```diff
@@ -54,48 +54,47 @@ func IncludeSharePointRestoreDataSelectors(
 	sel *selectors.SharePointRestore,
```

Comment:
L 53: Comment: `builds the common data-selector inclusions for SharePoint commands.` What are the data-selector inclusions for SharePoint commands?

Reply:
For categories: Libraries and Lists.  Also, SharePoint is a little unique in that it treats WebURL as a standard inclusion, rather than a filter, since the value is interchangeable (from the user's pov) with SiteIDs.